src/main/java/eu/dissco/backend/controller/MasJobRecordController.java

```diff
@@ -53,4 +54,11 @@ public ResponseEntity<JsonApiListResponseWrapper> getMasJobRecordsForCreator(
 
   }
 
+  @PatchMapping(value = "/{creatorId}/{masJobId}/running")
```

Comment:
We discussed this. Let's leave it this way for now, maybe in the future we can remove the creator (if each MAS get's it's own client token or something).

Reply:
noted in something to discuss further